accord-core/src/main/java/accord/coordinate/RecoverWithRoute.java

```diff
@@ -211,9 +211,15 @@ else if (!known.definition.isOrWasKnown())
                 if (known.executeAt.isDecidedAndKnownToExecute() && known.deps.hasDecidedDeps() && known.outcome == Apply)
                 {
                     Deps deps = full.stableDeps.reconstitute(route());
-                    node.withEpoch(full.executeAt.epoch(), () -> {
+                    node.withEpoch(full.executeAt.epoch(), (ignored, withEpochFailure) -> {
+                        if (withEpochFailure != null)
+                        {
+                            // TODO (review): We already ignore persist errors here by not providing a callback to persist
```

Comment:
same as above, shouldn't the `agent` be notified?

Reply:
I reworked this to have them all pass a wrapped exception to the agent.